modules/end-user-guide/pages/using-a-git-provider-access-token.adoc

```diff
@@ -59,7 +59,7 @@ If you have the link:https://www.gnu.org/software/coreutils/base64[base64] comma
 +
 * GitHub: link:https://docs.github.com/en/rest/users/users#get-a-user[Get a user]. See the `id` value in the response.
 * GitLab: link:https://docs.gitlab.com/ee/api/users.html#for-normal-users[List users: For normal users], use the `username` filter: `/users?username=:username`. See the `id` value in the response.
-* Bitbucket Server: link:https://developer.atlassian.com/server/bitbucket/rest/v802/api-group-api/#api-api-latest-users-userslug-get[Get users]. See the `id` value in the response.
+* Bitbucket Server: use the `curl` command: `curl -D- -u username:password -X GET -H "Content-Type: application/json" https://{baseurl}/rest/api/latest/users/{userSlug}` and see the `id` value in the response. More information could be found in the link:https://developer.atlassian.com/server/bitbucket/rest/v802/api-group-api/#api-api-latest-users-userslug-get[Get users BitBucket documentation] and the link:https://developer.atlassian.com/server/bitbucket/how-tos/example-basic-authentication/[authentication for BitBucket API].
```

Comment:
@svor,  in `https://{baseurl}/rest/api/latest/users/{userSlug}`,
`{baseurl}` and `{userSlug}` are some environment variables or are these placeholders for some values that the reader has to enter?

Reply:
Thank you.